08-mmc-upload/mmc-download.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+SCRIPTPATH="$( cd "$( dirname "${BASH_SOURCE[0]}"  )" && pwd  )"
+
+# Source FPGA mapping
+. ${SCRIPTPATH}/../misc/crate-fpga-mapping.sh
+
+MMC_BINARIES_DIR_REL=binaries
+MMC_BINARIES_DIR=${SCRIPTPATH}/${MMC_BINARIES_DIR_REL}
+
+OPENMMC_URL_RELEASE_BASE="https://github.com/lnls-dig/openMMC/releases/download"
+OPENMMC_VERSION="v1.6.0"
```

Comment:
openMMC version hard-coded here. I would make this script accept only version v1.6.0 and upwards (there is a simple way to do version string comparison in shell script?). Older versions are incompatible.

Reply:
Do you think that the version check should only be on this script or every script that receives the version value must check on this?

Reply:
Think just here is enough.